README.md

```diff
@@ -1,9 +1,10 @@
-<a href="https://github.com/cisagov/ScubaGear/releases" alt="ScubaGear version #">
-  <img src="https://img.shields.io/badge/ScubaGear-v1.2.0-%2328B953?labelColor=%23005288" />
-</a>
-<p>
+![CISA Logo](/docs/images/cisa.png)
 
-# ScubaGear
+<p>
+  <a href="https://github.com/cisagov/ScubaGear/releases" alt="ScubaGear version #">
+    <img src="https://img.shields.io/badge/ScubaGear-v1.2.0-%2328B953?labelColor=%23005288" />
```

Comment:
Should this link to v1.3.0? Or does that need to wait until after the release?

Reply:
Badge updated.

Reply:
I'm trying to find all the places that have 1.2 that should be 1.3.  Here's what I found:

* PowerShell/ScubaGear/ScubaGear.psd1 = someone has already mod'd this one
* PowerShell/ScubaGear/Sample-Reports/BaselineReports.html
  * There's link in here (line 493).  Should this be updated?  The URL it would link to does not exist.
  * Mod'd line 506.
* PowerShell/ScubaGear/Sample-Reports/TestResults.csv = Same a previous, lots of links that don't exist.  Should they be updated?
* Ugh, several other sample files with same issue.  

As a side note:  I can't believe this stuff is hard-coded in all these HTML file; it's too fragile.  There should be a way to automatically generate this stuff based on the value in ScubaGear.psd1.  Want me to create an issue for this?

Reply:
Ok, TY to Addam for clarifying that the scope of this is only the readme (Markdown) files.